completion/test_utils.py

```diff
@@ -16,6 +16,15 @@
 from . import waffle
 from .models import BlockCompletion
 
+def submit_completions_for_testing(user, course_key, block_keys):
```

Comment:
This refactor is nice. However, I believe in we can apply the refactor here into `completion/tests/test_models.py` as well.